### PR TITLE
Issue #1155 - upgrade to jakarta.json 2.0.1

### PR DESCRIPTION
**Pull request by JohnTimm**

### CODE REVIEW

File: fhir-model/src/main/java/com/ibm/fhir/model/parser/FHIRJsonParser.java [hunk 1]
```diff
@@ -39,14 +43,13 @@
...
 import com.ibm.fhir.model.util.ElementFilter;
-
 import net.jcip.annotations.NotThreadSafe;
 
 @NotThreadSafe
 @Generated("com.ibm.fhir.tools.CodeGenerator")
 public class FHIRJsonParser extends FHIRAbstractParser {
     public static boolean DEBUG = false;
-    private static final JsonReaderFactory JSON_READER_FACTORY = Json.createReaderFactory(null);
+    private static final JsonReaderFactory JSON_READER_FACTORY = Json.createReaderFactory(Collections.singletonMap(JsonConfig.REJECT_DUPLICATE_KEYS, true));
```
Comment (lmsurpre) [inline fhir-model/src/main/java/com/ibm/fhir/model/parser/FHIRJsonParser.java:52]: this line is what addresses #1155

Comment (JohnTimm) [reply fhir-model/src/main/java/com/ibm/fhir/model/parser/FHIRJsonParser.java:52]: Yes

**Review by lmsurpre — APPROVED** [review]
LGTM

**Review by prb112 — APPROVED** [review]
LGTM - also double checked the poms and this all looks good

**Pull request merged**